src/components/PageHeader.tsx

```diff
@@ -1,29 +1,72 @@
 import React from 'react'
 import { Container } from 'components/Container'
 import { makeStyles } from '@material-ui/core/styles'
+import { FIDER_LINK } from 'utils/constants'
 
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   root: {
-    maxHeight: 60
+    // minHeight: 65,
```

Comment:
I thought we are setting height here no?

Reply:
Not setting it makes it autosize.